### PR TITLE
Update Angular and Clarity to version 13

### PR DESCRIPTION
**Pull request by georgiatanasov147**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] Tests for the changes have been added (for bug fixes / features)
-   [ ] Examples have been added / updated (for bug fixes / features)
-   [ ] Changelog has been updated

## PR Type

What kind of change does this PR introduce?

-   [x] Bugfix
-   [ ] Feature
-   [ ] Code style update (formatting, local variables)
-   [x] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [x] CI related changes
-   [ ] Documentation content changes
-   [ ] Example website changes
-   [ ] Version bump
-   [ ] Other... Please describe:

## What does this change do?
- Updates the Angular and Clarity versions to 13

## What manual testing did you do?
- Ran npm start
- Made sure the application starts without any errors
- Looked at all components, checking  each of the Documentation/API/Examples tabs

## Testing done in VCD UI (with Angular and Clarity version 13 there as well)
- Installed local version of @vcd/i18n and @vcd/ui-components
- Executed all Unit Tests
- Executed top-routes smoke tests
- Tested plugins by installing the [customize-portal](https://gitlab.eng.vmware.com/cloud-director-solutions/customize-portal) and[ showcase-ui-plugin](https://gitlab.eng.vmware.com/cloud-director-solutions/showcase-ui-plugin) locally and checking all the extension points listed here https://vmware.github.io/vcd-ext-sdk/docs/development_examples/showcase_ui_plugin/#navigation-type-extension-points
- Tested branding by creating a custom theme with the old branding API and verifying its working 
- Started both provider and tenant projects without any errors
- Went through the main navigation screens on both tenant and provider

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/components/src/common/error/error-banner.component.spec.ts [hunk 1]
```diff
@@ -45,23 +45,23 @@ describe('ErrorBannerComponent', () => {
...
         });
 
-        it('is `status` for the alertType `warning`', function(this: HasFinderAndError): void {
+        it('is `status` for the alertType `warning`', function (this: HasFinderAndError): void {
             this.finder.hostComponent.errorBanner.alertType = 'warning';
             this.finder.detectChanges();
-            const errorBannerWO = this.finder.find({ woConstructor: ErrorBannerWidgetObject });
-            expect(errorBannerWO.ariaRole).toBe('status');
+            const errorBannerWO = this.finder.find<ErrorBannerWidgetObject<TestElement>>(ErrorBannerWidgetObject);
+            expect(errorBannerWO.ariaRole.attributes().getNamedItem('role')?.value).toBe('status');
```
[the remark this answers is not in the document]
Comment (juanmendes) [reply projects/components/src/common/error/error-banner.component.spec.ts:57]: We may have to chat as to why you moved this out of the widget object.

File: projects/components/package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
     "name": "@vcd/ui-components",
-    "version": "12.0.1-dev.1",
+    "version": "13.0.1-dev.1",
```
Comment (juanmendes) [inline projects/components/package.json:3]: Let's not bump this yet. We bump it when we actually publish it.

**Review by juanmendes — CHANGES_REQUESTED** [review]

File: projects/components/src/lib/directives/show-clipped-text.directive.ts [hunk 3]
```diff
@@ -119,8 +119,12 @@ const tip = {
         tip.clearHideTimeout();
         tip.hideTimeout = window.setTimeout(() => {
             tip.isMouseOver = false;
-            tip.container.style.opacity = '0';
-            tip.content.style.opacity = '0';
+            if (tip.container !== null) {
```
[the remark this answers is not in the document]
Comment (georgiatanasov147) [reply projects/components/src/lib/directives/show-clipped-text.directive.ts:122]: Tried that, but I was getting the following error: 
`TS2779: The left-hand side of an assignment expression may not be an optional property access.`
That's why I resorted to the if statement.

File: projects/components/src/common/error/error-banner.component.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (georgiatanasov147) [reply projects/components/src/common/error/error-banner.component.spec.ts]: I just wanted to update the tests not to use the old WidgetObject but it seems unnecessary. Will revert this and the changes to the widget object.

File: projects/components/src/common/error/error-banner.wo.ts [hunk 4]
```diff
@@ -3,19 +3,18 @@
...
-export class ErrorBannerWidgetObject extends WidgetObject<ErrorBannerComponent> {
+export class ErrorBannerWidgetObject<T> extends BaseWidgetObject<T> {
     static tagName = 'vcd-error-banner';
 
     /**
      * The ARIA role of the component.
      */
-    get ariaRole(): string {
-        return this.findElement('clr-alert').nativeElement.getAttribute('role');
+    get ariaRole(): T {
```
[the remark this answers is not in the document]
Comment (georgiatanasov147) [reply projects/components/src/common/error/error-banner.wo.ts]: I just wanted to update the tests not to use the old WidgetObject but it seems unnecessary. Will revert this and the changes to the widget object.

**Pull request merged**